ecs-logs-collector.sh

```diff
@@ -632,6 +633,27 @@ get_os_release(){
   fi
 }
 
+get_cgroupv2_events(){
+  # cgroup v2 is only supported on systemd systems
+  if [[ "$init_type" != "systemd" ]]; then
+    return 0
+  fi
+  # this file will only exist on systems with cgroup v2 enabled
+  if [ ! -f /sys/fs/cgroup/cgroup.controllers ]; then
+    return 0
+  fi
+  try "collect cgroupv2 events"
+
+  local outfile="${info_system}"/cgroupv2.events
+  touch $outfile
+  for eventfile in $(find /sys/fs/cgroup/ -type f -name "*.events" | grep ECSTask); do
+    echo "$eventfile" >> $outfile
+    cat $eventfile >> $outfile
+    echo "" >> $outfile
```

Comment:
We are appending the data of all the events file to one file? Is it possible that the files gets too big if a customer has a lot of task running? Can these files be separately added to the log folder like ECS Agent logs?

Reply:
do you mean the file is too large for text editors to handle? it's only a few hundred bytes per task so it's very unlikely this file would get too large